src/server/saved_objects/service/lib/repository.js

```diff
@@ -135,37 +135,46 @@ export class SavedObjectsRepository {
       ]), []),
     });
 
-    return items.map((response, i) => {
-      const {
-        error,
-        _id: responseId,
-        _version: version,
-      } = Object.values(response)[0];
-
-      const {
-        id = responseId,
-        type,
-        attributes,
-      } = objects[i];
+    return {
+      saved_objects: items.map((response, i) => {
```

Comment:
Realize it's weird to returning a snake cased object here. I see it's consistent with the existing code, so it doesn't need to be changed here, just want to bring it up.